api/services/okta_service.py

```diff
@@ -101,7 +112,7 @@ async def _list_users() -> list[User]:
             assert users is not None and resp is not None
 
             while resp.has_next():
-                more_users, _ = await resp.next()
+                more_users, _ = await OktaService._retry(resp.next())
```

Comment:
I believe we need to pass in the function instead of it's execution to `_retry()`

```suggestion
                more_users, _ = await OktaService._retry(resp.next)
```

Reply:
Oh good catch. Fixed it